### PR TITLE
Update docs for 2021.3 release

### PR DESCRIPTION
**Pull request by PetrovKP**

### CODE REVIEW

File: doc/sources/index.rst [hunk 1]
```diff
@@ -24,6 +24,14 @@ The acceleration is achieved through the use of the Intel(R) oneAPI Data Analyti
 (`oneDAL <https://github.com/oneapi-src/oneDAL>`_). Patching scikit-learn makes it a
 well-suited machine learning framework for dealing with real-life problems.
 
+.. image:: _static_static/scikit-learn-acceleration-2021.2.3.PNG
```
Comment (outoftardis) [inline doc/sources/index.rst:27]:
```suggestion
.. image:: _static/scikit-learn-acceleration-2021.2.3.PNG
```

try this

**Comment by PetrovKP** [comment]
[documentation (1).zip](https://github.com/intel/scikit-learn-intelex/files/6654198/documentation.1.zip)

File: README.md [hunk 2]
```diff
@@ -161,33 +161,7 @@ python runner.py --configs configs/blogs/skl_conda_config.json –report --no-in
...
-|| **TSNE**|All parameters except `metric` != 'euclidean' or `minkowski` with `p` != 2. | Sparse data is not supported. |
-|Unsupervised|**NearestNeighbors**|All parameters except `metric` != 'euclidean' or `minkowski` with `p` != 2. | Sparse data is not supported. |
-|Other|**train_test_split**|All parameters are supported. | Only dense data is supported.|
-||**assert_all_finite**|All parameters are supported. | Only dense data is supported. |
-||**pairwise_distance**|With `metric`='cosine' and 'correlation'.| Only dense data is supported. |
-||**roc_auc_score**|Parameters `average`, `sample_weight`, `max_fpr` and `multi_class` are not supported. | No limitations. |
-
- </details>
+Intel(R) Extension for Scikit-learn patching affects performance of specific Scikit-learn functionality.  Support list of algorithms and parameters see [here](https://intel.github.io/scikit-learn-intelex/algorithms.html). In cases when unsupported parameters are used, the package fallbacks into original Scikit-learn. If the patching does not cover your scenarios, [submit an issue on GitHub](https://github.com/intel/scikit-learn-intelex/issues).
```
Comment (outoftardis) [inline README.md:164]:
```suggestion
Intel(R) Extension for Scikit-learn patching affects performance of specific Scikit-learn functionality. Refer to the [list of supported algorithms and parameters](https://intel.github.io/scikit-learn-intelex/algorithms.html) for details. In cases when unsupported parameters are used, the package fallbacks into original Scikit-learn. If the patching does not cover your scenarios, [submit an issue on GitHub](https://github.com/intel/scikit-learn-intelex/issues).
```

File: doc/sources/algorithms.rst [hunk 3]
```diff
@@ -102,7 +114,7 @@ algorithms:
      - Only dense data is supported.
    * - Other
      - pairwise_distance
-     - With metric=``cosine`` and ``correlation``.
+     - With ``metric`` = 'cosine' and 'correlation'.
```
Comment (outoftardis) [inline doc/sources/algorithms.rst:117]:
```suggestion
     - With ``metric`` = 'cosine' or 'correlation'.
```

File: doc/sources/index.rst [hunk 4]
```diff
@@ -49,32 +57,45 @@ as the underlying solver, while getting the same solution faster.
...
     kmeans = KMeans(n_clusters=2, random_state=0).fit(X)
     print(f"kmeans.labels_ = {kmeans.labels_}")
-    pred = kmeans.predict([[0, 0], [12, 3]])
-    print(f"pred = {pred}")
-    print(f"kmeans.cluster_centers_ = {kmeans.cluster_centers_}")
 
 In the example above, you can see that the use of the original Scikit-learn
 has not changed. This behavior is achieved through drop-in patching.
 
 To undo the patch Scikit-learn with::
```
Comment (outoftardis) [inline doc/sources/index.rst:75]:
```suggestion
To undo the patch, run::
```

File: doc/sources/index.rst [hunk 5]
```diff
@@ -49,32 +57,45 @@ as the underlying solver, while getting the same solution faster.
...
 
     sklearnex.unpatch_sklearn()
+    # Need to re-import sklearn algorithms after the unpatch
```
Comment (outoftardis) [inline doc/sources/index.rst:78]:
```suggestion
    # You need to re-import scikit-learn algorithms after the unpatch:
```

File: doc/sources/index.rst [hunk 6]
```diff
@@ -49,32 +57,45 @@ as the underlying solver, while getting the same solution faster.
...
+Patching only one algorithm can be done as follows::
+
+    from sklearnex import patch_sklearn
+    # The names matched to scikit-learn estimators
+    patch_sklearn("SVC")
+
+Patching several algorithm can be done as follows::
+
+    from sklearnex import patch_sklearn
+    # The names matched to scikit-learn estimators
```
Comment (outoftardis) [inline doc/sources/index.rst:90]:
```suggestion
You may specify which algorithms to patch:

- Patching only one algorithm::

    from sklearnex import patch_sklearn
    # The names match scikit-learn estimators
    patch_sklearn("SVC")

- Patching several algorithms::

    from sklearnex import patch_sklearn
    # The names match scikit-learn estimators
```

I'd also suggest putting this info before the part with the unpatching.

**Review by outoftardis — APPROVED** [review]

**Comment by PetrovKP** [comment]
[documentation.zip](https://github.com/intel/scikit-learn-intelex/files/6658103/documentation.zip)

File: README.md [hunk 7]
```diff
@@ -106,6 +106,8 @@ pip install --upgrade dpcpp_cpp_rt
 conda install dpcpp_cpp_rt -c intel
 ```
 
+It's available as a part of [Intel oneAPI AI Analytics Toolkit](https://software.intel.com/content/www/us/en/develop/tools/oneapi/ai-analytics-toolkit.html#gs.3lkbv3) (AI Kit)
```
Comment (outoftardis) [inline README.md]:
```suggestion
The extension is available as a part of [Intel® oneAPI AI Analytics Toolkit](https://software.intel.com/content/www/us/en/develop/tools/oneapi/ai-analytics-toolkit.html) (AI Kit).
```

File: README.md [hunk 8]
```diff
@@ -47,14 +47,15 @@ You may reach out to project maintainers privately at onedal.maintainers@intel.c
 # 🛠 Installation
 Intel(R) Extension for Scikit-learn is available at the [Python Package Index](https://pypi.org/project/scikit-learn-intelex/),
 on Anaconda Cloud in [Conda-Forge channel](https://anaconda.org/conda-forge/scikit-learn-intelex) and in [Intel channel](https://anaconda.org/intel/scikit-learn-intelex).
+Also Intel(R) Extension for Scikit-learn is available available as a part of [Intel oneAPI AI Analytics Toolkit](https://software.intel.com/content/www/us/en/develop/tools/oneapi/ai-analytics-toolkit.html#gs.3lkbv3) (AI Kit)
```
Comment (outoftardis) [inline README.md:50]:
```suggestion
Intel(R) Extension for Scikit-learn is also available as a part of [Intel® oneAPI AI Analytics Toolkit](https://software.intel.com/content/www/us/en/develop/tools/oneapi/ai-analytics-toolkit.html) (AI Kit).
```

**Pull request merged**